cvc/tools/cvc_print.py

```diff
@@ -31,12 +31,58 @@
 logger = logging.getLogger(__name__)
 cert_dir = b''
 
+# Authorization bits according to
+# BSI-TR-03110-4 Chapter 2.2.3.2 Table 4
+AuthorizationBits = {
```

Comment:
It is not necessary to have a dict, a reverse list is sufficient.

Reply:
I like the explicit functionality to check set bits with human readable strings like `self.assertTrue(bits[AuthorizationBits["Age Verification"]])`

How would you structure the code with a reverse list ?